backend/src/services/TwilioCall.ts

```diff
@@ -114,6 +114,11 @@ class TwilioCallService extends TypedEventEmitter<
     const callEntity = await this.callService.getCallByIncomingSid(
       twilioCall.parentCallSid
     );
+    if (!callEntity) {
+      const msg = `callEntity not found for parentCallSid ${twilioCall.parentCallSid}`;
```

Comment:
Hm, is this prettier formatting?

Reply:
nope. mine.

Reply:
Let's put it on one line, don't see a need for this to be on two lines?

Reply:
what do u mean on one line? it is part of a block of code 
```    if (!callEntity) {
      const msg = `callEntity not found for parentCallSid ${twilioCall.parentCallSid}`;
      logger.error(msg);
      throw new Error(msg);
    }```

Reply:
I think I was tired when I was reviewing this, looks like the line break was from github.